ride_share.rb

```diff
@@ -0,0 +1,90 @@
+driver_data_for_use = [
```

Comment:
Hm, maybe instead of this variable name, I could imagine something like `trips_by_driver` or something like that, because the actual elements in the array represent trips